archive/imo/imo2022_q2.lean

```diff
@@ -0,0 +1,126 @@
+/-
+Copyright (c) 2022 boboquack. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: boboquack.
+-/
```

Comment:
What is the policy on using full names/handles in the copyright declaration?

Reply:
Is that a quesiton for me? (If so, I didn't see anything in the guidelines, but maybe I wasn't looking hard enough.)

Reply:
Yes but also for anyone who sees this question and has an opinion.
I would suppose that using full names are preferred and encouraged to say the least, but you may very well have reasons to oppose this, and I also do not know whether there is a strict guideline against breaking it.